action.yml

```diff
@@ -126,6 +126,7 @@ runs:
         &github_api_url=${{ github.api_url }}\
         &branch_name=${{ steps.branch-name.outputs.current_branch }}\
         &repo_name=${{ github.event.repository.name }}\
+        &pr_number=${{ github.event.pull_request.number }}\
```

Comment:
This is NOT intended to fix.
Just as a memo to clarify the spec of this feature:

It will run both `push` and `pull_request` event.
```yml
if: github.event_name == 'push' || github.event_name == 'pull_request'
```

Even if `push` event, seems to be able to access the property under `github.event.pull_request` (${{ github.event.pull_request.number }})
- test resutl: https://github.com/gizumon/github-actions-sandbox/actions/runs/4784357482/jobs/8505752328#step:6:11

The empty value in the request paramter would be treated as `0` in Golang, and will show `0` in the frontend.

```golang
PRNumber int `form:"pr_number" query:"pr_number" json:"pr_number"`
```

---

__summary:__

So that the `push` event won't overwrite the pull request number, we should recommend users to use `synchronize` type of `pull_request` event instead of `push` event.